support/checkstyle/src/main/resources/io/debezium/checkstyle/message.properties

```diff
@@ -1,4 +1,4 @@
 import.unused=Unused import - {0}.
 import.invalidJavaDocTag=Invalid JavaDoc tag - {0}
 header.missing=Missing a header - not enough lines in file. Every source file must contain the standard header; see 'checkstyle/src/main/resources/debezium.header'.
-header.mismatch=Line does not match expected header line of ''{0}''. Every source file must contain the standard header; see 'checkstyle/src/main/resources/debezium.header'.
\ No newline at end of file
+header.mismatch=Line does not match expected header line of ''{0}''. Every source file must contain the standard header; see 'checkstyle/src/main/resources/debezium.header'.
```

Comment:
This looks superfluous, could you revert this line ending modification?

Reply:
Sure, done.